src/Standards/Generic/Sniffs/CodeAnalysis/EmptyPHPStatementSniff.php

```diff
@@ -53,10 +53,6 @@ public function process(File $phpcsFile, $stackPtr)
         case 'T_SEMICOLON':
             $prevNonEmpty = $phpcsFile->findPrevious(Tokens::$emptyTokens, ($stackPtr - 1), null, true);
 
-            if ($prevNonEmpty === false) {
```

Comment:
Could the same condition on line 127 be removed as well ?

Reply:
I believe so, just removed it in a new commit.